maintainers/maintainer-list.nix

```diff
@@ -3,12 +3,12 @@
     handle = {
       # Required
       name = "Your name";
+      github = "GithubUsername";
+      githubId = your-github-id;
```

Comment:
Might be good to link to how you can find this. I don't think I ever had to deal with my GitHub ID before contributing to nixpkgs.

Reply:
You can find it just below with multiple examples I believe.

Reply:
We already have documentation on why it may be preferable for some to add GitHub data: https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L29

This documentation could be extended if there are more benefits to expected when adopting GitHub, like the security tracker you mentioned. This allows contributors to make an informed decision on whether this outweighs the downsides.

I see no point in adding more mandatory fields. It's risky in the first place to put personal data on storage that is meant to be immutable. So, any means to keep it to a minimum should be considered.